.bitmap

```diff
@@ -457,12 +463,24 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/webpack/config-mutator"
     },
+    "modules/create-element-from-string": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
+        "rootDir": "scopes/html/utils/create-element-from-string"
```

Comment:
change all the paths from 
`scopes/html/utils` to `scopes/html/modules` 
This is our standard across the repo.
(you can just do `bit move` for each one of them

Reply:
Done